docs/site/api.md

```diff
@@ -141,6 +141,7 @@ Example: [/api/v2/contest/bts19](https://dmoj.ca/api/v2/contest/bts19)
     "has_rating": "<whether the contest has been rated>",
     "rating_floor": "<the minimum user rating required for the user to be rated>",
     "rating_ceiling": "<the maximum user rating for the user to be rated>",
+    "performance_ceiling": "<the maximum user rating possible from this contest>",
```

Comment:
Do you document that it will be `None` if there's no performance ceiling set?

Reply:
Done